content/13.conclusion.md

```diff
@@ -1 +1,6 @@
 ## Conclusion {#conclusion}
+
+Deep learning techniques have the potential for wide use in biology, meeting or exceeding the performance of both humans and the current state-of-the art algorithms in a variety of tasks while generating biological insights that may be applied back to fundamental research. 
```

Comment:
This is a long sentence. Can we break up?

Reply:
Done!